src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -857,7 +868,7 @@ private function findAdderAndRemover(\ReflectionClass $reflClass, array $singula
      *
      * @return bool Whether the method is public and has $parameters required parameters
      */
-    private function isMethodAccessible(\ReflectionClass $class, $methodName, $parameters)
+    private function isMethodAccessible(\ReflectionClass $class, $methodName, $parameters, \ReflectionMethod &$method = null)
```

Comment:
Not sure if it's really is a good idea to make the method have this side effect. I think I would rather split it to extract the part that is getting the `ReflectionMethod` instance.